kong/db/declarative/import.lua

```diff
@@ -289,13 +307,20 @@ local function _set_entity_for_txn(t, entity_name, item, options, is_delete)
   -- store serialized entity into lmdb
   t:set(itm_key, itm_value)
 
+  -- for global query
+  local global_key = item_key(entity_name, GLOBAL_WORKSPACE_TAG, pk)
+  t:set(global_key, idx_value)
+
   -- select_by_cache_key
   if schema.cache_key then
     local cache_key = dao:cache_key(item)
-    local key = unique_field_key(entity_name, ws_id, "cache_key", cache_key)
 
-    -- store item_key or nil into lmdb
-    t:set(key, idx_value)
+    for _, wid in ipairs {ws_id, GLOBAL_WORKSPACE_TAG} do
+      local key = unique_field_key(entity_name, wid, "cache_key", cache_key)
+
+      -- store item_key or nil into lmdb
+      t:set(key, idx_value)
```

Comment:
not a blocker, just a comment for other reviewers. TODO: we only want to save the entity with key containing specific ws_id, not a * ws_id, which does not consume too much memory space for lmdb. https://konghq.atlassian.net/browse/KAG-5704